### PR TITLE
fix: replace certbot-auto with certbot

### LINKED ISSUE #1105: certbot-auto deprecated
Issue: Bug report

Ubuntu 20.04.1 LTS and Ubuntu 18.04.5 LTS

bench version
erpnext 13.x.x-develop
frappe 13.x.x-develop
I just did a bench update as well 

I followed the Guide on 
https://frappeframework.com/docs/user/en/bench/guides/lets-encrypt-ssl-setup

```
sudo -H bench setup lets-encrypt domain.com
Running this will stop the nginx service temporarily causing your sites to go offline
Do you want to continue? [y/N]: y
$ sudo systemctl stop nginx
$ /opt/certbot-auto  --config /etc/letsencrypt/configs/domain.com.cfg certonly
Skipping bootstrap because certbot-auto is deprecated on this system.
Your system is not supported by certbot-auto anymore.
Certbot cannot be installed.
Please visit https://certbot.eff.org/ to check for other alternatives.
nginx.conf already exists and this will overwrite it. Do you want to continue? [y/N]: n
```

**Possible Solution** :bookmark_tabs:
I followed this guide here and was able to install the cert. It does install certbot instead of certbot-auto but the command otherwise is the same.

### LINKED ISSUE #1206: bench setup lets-encrypt [site] fails due to 404 when downloading certbot-auto
Issue: Bug report

**Describe the bug** :chart_with_downwards_trend:
`bench setup lets-encrypt` fails due to a 404 when fetching certbot-auto

During the lets-encrypt install process, the URL bench uses to retrieve `certbot-auto` is no longer available and produces a 404 (https://dl.eff.org/certbot-auto): 
https://github.com/frappe/bench/blob/8b0d522b639575e0fd651ddcd5cd5ab6b91d43c4/bench/config/lets_encrypt.py#L109

The failure causes `bench setup lets-encrypt` to fail without producing output. Examining `bench/logs/bench.log` shows output similar to the following:

```
2021-10-18 20:49:00,619 DEBUG sudo systemctl stop nginx
2021-10-18 20:49:00,683 WARNING /usr/local/bin/bench setup lets-encrypt [SITE] executed with exit code 404
```

**Expected behavior** :chart_with_upwards_trend:
`bench setup lets-encrypt` should succeed or log that it failed in stderr.

**OS (please complete the following information):** :cyclone:
- [ ] Linux: `ubuntu:20.04`

**Version Information**

Can be found out by running `bench version` in your respective bench folder.
- Bench Branch: see https://github.com/frappe/bench/blob/8b0d522b639575e0fd651ddcd5cd5ab6b91d43c4/bench/config/lets_encrypt.py#L109

**Additional context** :page_facing_up:
Related: https://github.com/frappe/bench/issues/1105

### LINKED ISSUE #1301: Certbot auto script is deprecated
According to https://community.letsencrypt.org/t/cant-download-file-in-https-dl-eff-org-certbot-auto/159053
This script is deprecated. Therefor, causing `bench setup lets-encrypt` to crash.

**Usage**
https://github.com/frappe/bench/blob/eb715735bbb79e5edd39e29c1b0e5587d2d3712c/bench/config/lets_encrypt.py#L117

### PR DESCRIPTION
**Pull request by dj12djdjs**
Continuation of: https://github.com/frappe/bench/pull/1152

closes: https://github.com/frappe/bench/issues/1301
closes: https://github.com/frappe/bench/issues/1105
closes: https://github.com/frappe/bench/issues/1206

TODO:

- [x] Implement certbot install instructions 
https://github.com/frappe/bench/pull/1152#discussion_r672080631
- [x] Test

Proof:
1. `sudo bench setup lets-encrypt SITE_NAME`
image
We receive error to install via: https://certbot.eff.org/instructions

2. Input your settings
image
Then follow certbot install instructions.

3. Follow CLI instructions
image

### CODE REVIEW

File: bench/config/templates/frappe_sudoers [hunk 1]
```diff
@@ -15,6 +15,6 @@
 {{ user }} ALL = (root) NOPASSWD: {{ nginx }}
 {% endif %}
 
-{{ user }} ALL = (root) NOPASSWD: /opt/certbot-auto
+{{ user }} ALL = (root) NOPASSWD: /usr/bin/certbot
```
Comment (gavindsouza) [inline bench/config/templates/frappe_sudoers]: Shouldn't we use `get_certbot_path` to fetch the path for this too?

**Comment by gavindsouza** [comment]
hey @dj12djdjs, I don't have any means to confirm this works right now. Can you share screens & environment details for your test setup?

**Comment by dj12djdjs** [comment]
> hey @dj12djdjs, I don't have any means to confirm this works right now. Can you share screens & environment details for your test setup?

Sure, I'll try to submit the proof this weekend.

**Comment by gavindsouza** [comment]
> > hey @dj12djdjs, I don't have any means to confirm this works right now. Can you share screens & environment details for your test setup?
> 
> Sure, I'll try to submit the proof this weekend.

Ping

**Comment by dj12djdjs** [comment]
> > > hey @dj12djdjs, I don't have any means to confirm this works right now. Can you share screens & environment details for your test setup?
> > 
> > 
> > Sure, I'll try to submit the proof this weekend.
> 
> Ping

Sorry for the delay, I should be able to spin up a server tomorrow and send some screen shots.

**Comment by dj12djdjs** [comment]
@gavindsouza Screen shots posted

**Review by gavindsouza — APPROVED** [review]

**Pull request merged**